### ISSUE TITLE
♿Time Picker AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1345]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
[Text labels should come before the input element
](https://aat.w3ibm.mybluemix.net/token/b62ec93e-0a1f-4e82-b96a-18e75818d205/51d217d2-9b4e-4e3e-a04e-66809340d9d0/latest///doc/w3/help/en-US/idhi_accessibility_check_g1017.html)<img width="595" alt="screen shot 2018-10-25 at 5 02 21 pm" src="https://user-images.githubusercontent.com/40970507/47533046-d7524100-d877-11e8-8646-139ecb7ce19b.png">
## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset

**Comment by alisonjoseph** [comment]
We already fixed this in the React library, so should be as simple as updating the html

**Comment by tw15egan** [comment]
Will fix when I update Time Picker (waiting on #1348)
